packages/tpetra/core/src/Tpetra_Distributor.hpp

```diff
@@ -1419,6 +1545,13 @@ namespace Tpetra {
         "The \"send buffer\" code path doesn't currently work with "
         "nonblocking sends.");
 
+      // FIXME (CAG 01 April 2019) This does not work for Persistent.
```

Comment:
Why not?  Is this because persistent sends are nonblocking, and thus share the same issues as the `MPI_Isend` code path?

Reply:
Yep.